src/sidebars/favourites/tearout-directive.js

```diff
@@ -160,6 +161,10 @@
                         function boundsChangingEvent() {
                             if (insideFavouritesPane()) {
                                 reorderFavourites();
+                                dragService.setOtherInstance(null);
```

Comment:
Outside of the drag service all the calls to this method pass `null`. Further to that I don't think anything outside of the class should be able to specify another instance. I think I'd prefer a `clearOtherInstance` to start with, but I'll have a think of whether I prefer another pattern.